app/src/main/java/agency/five/codebase/android/movieapp/ui/home/di/HomeModule.kt

```diff
@@ -4,13 +4,14 @@ import agency.five.codebase.android.movieapp.ui.home.HomeViewModel
 import agency.five.codebase.android.movieapp.ui.home.mapper.HomeScreenMapper
 import agency.five.codebase.android.movieapp.ui.home.mapper.HomeScreenMapperImpl
 import org.koin.androidx.viewmodel.dsl.viewModel
+import org.koin.core.qualifier.named
 
 import org.koin.dsl.module
 
 val homeModule = module {
     viewModel {
         HomeViewModel(
-            movieRepository = get(),
+            movieRepository = get(named("dispatcherIo")),
```

Comment:
good practice is to extract values like `dispatcherIo` to constant in the right place (concurrencyModule) and use the constant. This is error prone because of typos